.changeset/calm-bulldogs-repair.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-docs/gatsby-theme-docs': patch
```

Comment:
nitpick: this is not a patch but a minor release (a new feature).   

If I'm overzealous it's even a major because the API of the component changed.

Reply:
I changed it to a minor now 👍